src/main/kotlin/lotto/domain/LottoTicket.kt

```diff
@@ -14,7 +15,17 @@ value class LottoTicket private constructor(val numbers: List<Int>) {
         private const val ONE_THOUSAND = 1000
 
         fun generateByAuto(): LottoTicket {
-            return LottoTicket((1..45).shuffled().subList(0, 6).sorted())
+            return LottoTicket(
+                (LottoNumber.MINIMUM_NUMBER..LottoNumber.MAXIMUM_NUMBER)
+                    .shuffled()
+                    .subList(0, 6)
+                    .sorted()
+                    .map(LottoNumber.Companion::from)
+            )
+        }
+
+        fun generateByManual(numbers: List<Int>): LottoTicket {
+            return LottoTicket(numbers.map(LottoNumber.Companion::from))
```

Comment:
`Companion` keyword는 의도하신걸까요??

```suggestion
            return LottoTicket(numbers.map(LottoNumber::from))
```

Reply:
아닙니다. 클래스 이름으로 참조 접근할 수 있는지 몰랐습니다! 오늘도 배워갑니다:)